packages/cli/__tests__/zosfiles/__integration__/edit/ds/__snapshots__/edit.ds.integration.test.ts.snap

```diff
@@ -148,11 +157,16 @@ exports[`Edit Data Set should display the help 1`] = `
 
       $ zowe zos-files edit data-set ibmuser.cntl(iefbr14) --editor C:\\\\Windows\\\\System32\\\\Notepad.exe
 
-   - Edit the contents of the USS file \\"/a/ibmuser/my_jcl.jcl\\"
-   in notepad and set extension as \\"jcl\\":
+   - Edit the contents of the USS file \\"ibmuser.jcl(iefbr14)\\"
+   and set extension as \\"jcl\\":
 
       $ zowe zos-files edit data-set ibmuser.jcl(iefbr14) --editor notepad --extension jcl
 
+   - Edit the contents of the USS file \\"ibmuser.jcl(iefbr14)\\"
+   and set extension as \\"jcl\\":
+
+      $ zowe zos-files edit data-set ibmuser.jcl(iefbr14) --editor notepad --binary
```

Comment:
The example text for the `edit ds` command shouldn't mention USS files.

Commenting on the snapshots just because I noticed it here when reviewing the PR. The place to update would likely be in `-strings-/en.ts`, and then we can have jest rebuild these snapshots.